__tests__/app.test.js

```diff
@@ -57,10 +57,30 @@ describe("GET/api/categories", () => {
         });
     });
   })
+  describe("GET /api/reviews/:review_id", () => {
+    test("should return a review object with the specified properties", () => {
+      return request(app)
+        .get("/api/reviews/2")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body).toEqual({
+            review_id: 2,
+            title: expect.any(String),
+            review_body: expect.any(String),
+            designer: expect.any(String),
+            review_img_url: expect.any(String),
+            votes: expect.any(Number),
+            category: expect.any(String),
+            owner: "philippaclaire9",
+            created_at: expect.any(String),
+          });
+        });
+    });
+  })
```

Comment:
Still need your sad path tests in here
- 404 article not found
- 400 article id invalid